templates/SilverStripe/CMS/Controllers/Includes/CMSMain_PageList_Sidebar.ss

```diff
@@ -1 +1 @@
-<% include SilverStripe\\CMS\\Controllers\\CMSMain_TreeView_Deferred %>
+<% include SilverStripe\\CMS\\Controllers\\CMSMain_PageList %>
```

Comment:
Are we not deferring the tree view anymore?

Reply:
I'd removed deferred intentionally earlier, but was requested to put it back because it increased the overall "until editable" wait for page edit forms.

Reply:
@tractorcow You are still deferring tree view and as well as list view. We load either tree or list view dynamically via JS using the urls provided in `CMSMain_PageList.ss`.